Client/Frontend/Widgets/SnackBar.swift

```diff
@@ -173,24 +115,33 @@ class SnackBar: UIView {
     }
 
     private func setupButtonViewLayout(constant: CGFloat = 0) {
-        buttonsView.snp.remakeConstraints { make in
-            make.bottom.equalTo(self.snp.bottom).inset(constant)
-            make.leading.trailing.equalTo(self)
-            if !self.buttonsView.subviews.isEmpty {
-                make.height.equalTo(UIConstants.SnackbarButtonHeight)
-            } else {
-                make.height.equalTo(0)
-            }
-        }
+        NSLayoutConstraint.deactivate(buttonsViewConstraints)
+        buttonsViewConstraints = [
+            buttonsView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -constant),
+            buttonsView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            buttonsView.trailingAnchor.constraint(equalTo: trailingAnchor)
+        ]
+        buttonsViewConstraints.append(!buttonsView.subviews.isEmpty ?
+                                      buttonsView.heightAnchor.constraint(equalToConstant: UIConstants.SnackbarButtonHeight) :
+                                        buttonsView.heightAnchor.constraint(equalToConstant: 0))
+        NSLayoutConstraint.activate(buttonsViewConstraints)
+    }
+
+    private func remakeScrollViewConstraints() {
```

Comment:
I think we don't need this method. The only constraint that can change here is the height. We could adjust it when `viewWillTransition(to:with:)` and `traitCollectionDidChange(_:)` is called by saving the height constraint in a var and updating the constant in these cases.